Sources/StreamChat/Workers/DatabaseCleanupUpdater_Tests.swift

```diff
@@ -6,14 +6,14 @@
 @testable import StreamChatTestTools
 import XCTest
 
-final class ChannelListCleanupUpdater_Tests: StressTestCase {
+final class DatabaseCleanupUpdater_Tests: StressTestCase {
     typealias ExtraData = NoExtraData
         
     var database: DatabaseContainerMock!
     var webSocketClient: WebSocketClientMock!
     var apiClient: APIClientMock!
     
-    var channelListCleanupUpdater: ChannelDatabaseCleanupUpdater<ExtraData>?
+    var channelListCleanupUpdater: DatabaseCleanupUpdater<ExtraData>?
```

Comment:
This property's name has dodged the renaming wave

Reply:
Right, will fix. Thanks!